src/btree/bt_split.c

```diff
@@ -1673,25 +1675,26 @@ __split_multi_inmem_fail(WT_SESSION_IMPL *session, WT_PAGE *orig, WT_MULTI *mult
  *     Move a multi-block entry into a WT_REF structure.
  */
 int
-__wt_multi_to_ref(WT_SESSION_IMPL *session, WT_PAGE *page, WT_MULTI *multi, WT_REF **refp,
-  size_t *incrp, bool closing)
+__wt_multi_to_ref(WT_SESSION_IMPL *session, WT_REF *old_ref, WT_PAGE *page, WT_MULTI *multi,
+  WT_REF **refp, size_t *incrp, bool first, bool closing)
```

Comment:
When should this `first` argument be set? I think that's worth a comment.

Reply:
first is the first page of the split page. We need to ensure it uses the same key in the internal page as the page before the split.